dashboard/src/actions/operators.ts

```diff
@@ -214,6 +214,26 @@ export function deleteResource(
   };
 }
 
+export function updateResource(
+  namespace: string,
+  apiVersion: string,
+  resource: string,
+  name: string,
+  body: object,
+): ThunkAction<Promise<boolean>, IStoreState, null, OperatorAction> {
+  return async dispatch => {
+    dispatch(creatingResource());
+    try {
+      const r = await Operators.updateResource(namespace, apiVersion, resource, name, body);
+      dispatch(resourceCreated(r));
```

Comment:
Should this be a `resourceUpdated(r)` action?

Reply:
I didn't want to add more boilerplate code but yes, it could be confusing